core/execution/future/market.go

```diff
@@ -5451,6 +5468,10 @@ func (m *Market) SubmitAMM(ctx context.Context, submit *types.SubmitAMM, determi
 
 	// create the AMM curves but do not confirm it with the engine
 	var order *types.Order
+	if err := VerifyAMMBounds(submit.Parameters.Base, submit.Parameters.LowerBound, submit.Parameters.UpperBound, m.priceFactor); err != nil {
```

Comment:
can you through the same check in `AmendAMM()` too?

Reply:
added